style.css

```diff
@@ -498,6 +508,58 @@
   border-bottom: #6f6c6b 1px solid;
 }
 
+.pop-up-general-container {
+  left: 5%;
+  right: 5%;
+  bottom: 5%;
+  top: 5%;
+  padding-top: 16px;
+  padding-bottom: 20px;
+  align-self: center;
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+  width: 90vw;
+  height: 90vh;
+  gap: 24px;
+  z-index: 300;
+  border: 1px solid #dfe1e6;
+  overflow-y: scroll;
+}
+
+.pop-up-general-container::-webkit-scrollbar {
+  display: none;
+}
+
+.pop-up-img-and-content {
+  gap: 24px;
+  max-height: 78%;
+}
+
+.contain {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+}
+
+.pop-up-close {
+  padding: 10px;
+  display: inline-flex;
+  align-self: flex-start;
+  cursor: pointer;
```

Comment:
- [ ] Please make your background on the navigation bar fit the whole screen (`height:100vh`) so that users don't see what's behind because it can distract them 😉 

![image](https://user-images.githubusercontent.com/90222110/190730420-880ada09-05ac-40ac-9a24-1dfaf17bdf88.png)

Reply:
ok